src/m365/spo/commands/web/web-applicationcustomizer-add.spec.ts

```diff
@@ -0,0 +1,126 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { telemetry } from '../../../../telemetry';
+import auth from '../../../../Auth';
+import { Cli } from '../../../../cli/Cli';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import { pid } from '../../../../utils/pid';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+const command: Command = require('./web-applicationcustomizer-add');
+import * as SpoCustomActionAddCommand from '../customaction/customaction-add';
+
+
+describe(commands.WEB_APPLICATIONCUSTOMIZER_ADD, () => {
+  const webUrl = 'https://contoso.sharepoint.com';
+  const title = 'PageFooter';
+  const clientSideComponentId = '76d5f8c8-6228-4df8-a2da-b94cbc8115bc';
+  const clientSideComponentProperties = '';
+
+
+  let log: any[];
+  let logger: Logger;
+  let commandInfo: CommandInfo;
+  let loggerLogToStderrSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+    auth.service.connected = true;
```

Comment:
We're missing here: `sinon.stub(pid, 'getProcessName').callsFake(() => '');`. We should include it to avoid unnecessary resolving of pid for the test runner